EncounterMe/EncounterMe/User.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Security.Cryptography;
+using System.Linq;
+
+namespace EncounterMe
+{
+    class User
+    {
+        private string name;
```

Comment:
You should probably move the variable declarations to the top of the class for clarity